src/openrct2/ride/RideRatings.cpp

```diff
@@ -898,12 +898,8 @@ static uint16_t ride_compute_upkeep(RideRatingUpdateState& state, Ride* ride)
     }
 
     // Add maintenance cost for reverser track pieces
-    uint16_t reverserMaintenanceCost = 80;
-    if (ride->type == RIDE_TYPE_REVERSER_ROLLER_COASTER)
-    {
-        reverserMaintenanceCost = 10;
-    }
-    upkeep += reverserMaintenanceCost * state.AmountOfReversers;
```

Comment:
No ride even has access to reversers beside the reverser roller coaster, so I don’t understand why we even have this check. The field also does not double up as something else.

You can just change `reverserMaintenanceCost` to be always 10 and remove the ride type check.

Reply:
I think it's still a good idea to expose the cost for each reverser piece in the RideTypeDescriptor so it's not hardcoded to 10.

Reply:
I disagree. This is consistent with other types of track, like brakes. _If_ there is ever a need for a field like that, we will add it. No need to bloat the code.